tests/acceptance/new-note-test.js

```diff
@@ -0,0 +1,17 @@
+import { test } from 'qunit';
+import moduleForAcceptance from 'remember/tests/helpers/module-for-acceptance'
+
+moduleForAcceptance('Acceptance | new note')
+
+  test('should add a new note on submit with valid input', function(assert) {
+    visit('reminders/new');
+    fillIn('.spec-input-title', 'Banana');
```

Comment:
I see you're pulling code from the groceries app. Doesn't seem like content that a reminder would contain. I'm fine with you using other projects as an example but make sure you aren't copy/pasting.